impeller/entity/contents/atlas_contents.cc

```diff
@@ -57,6 +64,85 @@ void AtlasContents::SetCullRect(std::optional<Rect> cull_rect) {
   cull_rect_ = cull_rect;
 }
 
+struct AtlasBlenderKey {
+  Color color;
+  Rect rect;
+
+  struct Hash {
+    std::size_t operator()(const AtlasBlenderKey& key) const {
+      return fml::HashCombine(key.color.red, key.color.green, key.color.blue,
```

Comment:
This is probably safe

Reply:
thinking about this more, there could be some inconsistencies with the hash function and equality implementation given that the latter uses nearly equal. Instead i'll keep the int representation of the color around and use that for both the hash and equality.